### PR TITLE
DDL classes for more updates

### PR DESCRIPTION
**Pull request by jhinkey**
Please see my notes in email.

### CODE REVIEW

**Comment by marcellustavares** [comment]
thanks @jhinkey @rbohl!

**Comment by rbohl** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-service/src/main/java/com/liferay/dynamic/data/lists/service/impl/DDLRecordLocalServiceImpl.java [hunk 1]
```diff
@@ -288,13 +295,15 @@ public void deleteRecord(long recordId) throws PortalException {
 	}
 
 	/**
-	 * Disassociates a record from the specified locale.
+	 * Disassociates the locale from record.
```
Comment (rbohl) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-service/src/main/java/com/liferay/dynamic/data/lists/service/impl/DDLRecordLocalServiceImpl.java]: @jhinkey is this correct or should we use "the record" or "a record"?

Comment (jhinkey) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-service/src/main/java/com/liferay/dynamic/data/lists/service/impl/DDLRecordLocalServiceImpl.java]: Good point. It should be "the record". The record ID parameter implies a specific record, for which we can refer to using a definite article. Thanks!

**Pull request closed without merging**